packages/matchbox/src/components/Grid/styles.js

```diff
@@ -0,0 +1,103 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+const gutterWidth = tokens.spacing_400;
+export const breakpoints = ['xs', 'sm', 'md', 'lg', 'xl'];
+const alignmentStyleMap = {
+  start: {
```

Comment:
I used objects to represent the style because it was easier to compose together.